calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/readers/CSVReader.java

```diff
@@ -176,6 +177,7 @@ protected String[] parseLineFromDelimitedFile(String line)
 				DELIMITER_TOKEN_IN_REGEX,
 				getHexStringForDelimiter());
 
+		line = CSVReader.removeUTF8BOM(line);
```

Comment:
what about if someone uploaded a file with some other Byte Order Mark, probably a good idea to handle other BOM types as well?
see https://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/input/BOMInputStream.html

Reply:
Done!